tests/__init__.py

```diff
@@ -160,3 +160,31 @@ def __exit__(self, *args, **kwargs):
         # Clean up by setting sys.stdout back to what it was before we opened
         # up this context
         sys.stdout = self._stdout
+
+class SwapStdin(object):
+    """Utility object using a context manager to swap out stdin with user-provided data."""
+    def __init__(self, *, input=None):
```

Comment:
`input` is a builtin, you may want to consider renaming the argument

Reply:
Forgot to do that, fixed!